state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -731,6 +748,8 @@ pub mod pallet {
 		PrivateChannelExistsForBroker,
 		/// The Broker does not have an open private channel.
 		NoPrivateChannelExistsForBroker,
+		/// The Bonder does not have enough Funds to cover the bond.
+		InsufficientFunds,
```

Comment:
This is unused. (Although i think we can use it instead of defining a new error in the flip pallet)